src/patterns/components/inputs/collection.yaml

```diff
@@ -39,8 +39,141 @@ restrictions:
   - You should use helper text, as linked below, when a user needs further clarification beyond the label.
   - Labels and error messages should be short and to the point.
   - Labels should not be longer than what will fit on one line at 320px.
-  - The data-id property is provided as a hook for automated tools. If you have multiple 
-    instances of the same variant of a component on the same page, make sure each instance 
+  - The data-id property is provided as a hook for automated tools. If you have multiple
+    instances of the same variant of a component on the same page, make sure each instance
     has a unique data-id property ("input-text-1", "input-text-2", "input-date-1", etc).
 sparkPackageCore: true
 hasAngularInfo: true
+variableTable:
+  $input-gray-medium:
+    default: #5a5a5a
+    description: The gray used throughout inputs.
```

Comment:
Should this be more generalized?

$input-primary-color:
     default: #5a5a5a
     description: The primary color used for borders on inputs